olca-app/src/org/openlca/app/editors/processes/data_quality/DataQualityCell.java

```diff
@@ -80,11 +94,13 @@ public void mouseHover(MouseEvent e) {
 	}
 
 	void setColor() {
-		Color color = null;
+		Color color;
 		if (selected)
```

Comment:
Please use braces around such if-else-blocks (exceptions are early returns, breaks, continues), another option is to write it like this (also see the message above above):

```java
var color = selected
  ? DQUI.getColor(...)
  : Colors.background();
```